docs/howtos/traffic-manager-permissions-setup.md

```diff
@@ -0,0 +1,53 @@
+# How-to Guide: Traffic Manager Permissions setup
+
+This guide provides an overview of how to set up permissions for Azure Traffic Manager in order to use the DNS based global
+load balancing feature.
+
+## Get the hub and member agents identity
+
+Figure out the identity used by hub-net-controller-manager and member-net-controller-manager.
+There are various ways to set up the fleet networking, and the recommended way is to use [workload-identity](https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster).
+
+```bash
+export HUB_IDENTITY_PRINCIPAL_ID=$(az identity show \
+    --name "${HUB_USER_ASSIGNED_IDENTITY_NAME}" \
+    --resource-group "${RESOURCE_GROUP}" \
+    --query principalId \
+    --output tsv)
+export MEMBER_IDENTITY_PRINCIPAL_ID=$(az identity show \
+    --name "${MEMBER_USER_ASSIGNED_IDENTITY_NAME}" \
+    --resource-group "${RESOURCE_GROUP}" \
+    --query principalId \
+    --output tsv)
+```
+
+## Create the role assignment for the hub agent
+
+### Create the role assignment for the hub agent to manage the Azure Traffic Manager
+Assign role “Traffic Manager Contributor” to hub cluster identity at the Azure Traffic Manager resource group scope
+```bash
+az role assignment create --assignee "${HUB_IDENTITY_PRINCIPAL_ID}" --role "a4b10055-b0c7-44c2-b00f-c7b5b3550cf7" --scope "/subscriptions/mySubscriptions/resourceGroups/MyAzureTrafficManagerResourceGroup"

```

Comment:
I think we can use the role name instead of the uid in the command? Using the role name is easier to understand.

Reply:
what is this role id pointing to?

Reply:
The role name can be renamed, though rename is a breaking change. Only the ID is persistent.

Added a doc link about the role.